pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -321,6 +321,7 @@ func (t TerraformDirectoryLoader) buildUnifiedConfig(rootMod *hclConfigs.Module,
 			// figure out path sub module directory, if it's remote then download it locally
 			var pathToModule string
 			var err error
+			var moduleDirDiags hcl.Diagnostics
 			if downloader.IsLocalSourceAddr(req.SourceAddr) {
```

Comment:
Wondering, if it is time to get rid of this if-else ladder and move to a switch case?